src/zzz_od/application/world_patrol/operation/world_patrol_run_route.py

```
@@ -117,112 +139,192 @@ def run_op(self) -> OperationRoundResult:
                 self.no_pos_start_time = self.last_screenshot_time
-            elif no_pos_seconds > 20:
-                return self.round_fail(status='无法计算坐标')
-            elif no_pos_seconds > 4:
-                # 无法计算坐标超过4秒，执行一次脱困；若达到尝试上限则触发当前路线重启
-                if self._get_rid_of_stuck():
-                    return self.round_fail(status='卡住超限，重启当前路线')
-            elif no_pos_seconds > 2:
-                self.ctx.controller.stop_moving_forward()
+            else:
+                if time_since_last_pos > self.TIME_THRESHOLD_FOR_FAIL_NO_POS:
+                    log.error('长时间无法计算坐标，任务失败')
```

Ruff (0.14.1)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)